pkg/clusters/addons/kong/kong.go

```diff
@@ -146,17 +166,39 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		)
 	}
 
+	if err := clusters.CreateNamespace(ctx, cluster, a.namespace); err != nil {
+		return err
+	}
+
 	// do the deployment and install the chart
 	args := []string{"--kubeconfig", kubeconfig.Name(), "install", DefaultDeploymentName, "kong/kong"}
-	args = append(args, "--create-namespace", "--namespace", a.namespace)
-	args = append(args, a.deployArgs...)
+	args = append(args, "--namespace", a.namespace)
+	if a.enterprise {
+		if err := deployKongEnterpriseLicenseSecret(ctx, cluster, a.namespace, KongLicenseSecretName); err != nil {
```

Comment:
If we're hardcoding `KongLicenseSecretName` here, what are we doing with the `b.license` value that was provided? It seems to just be ignored at this point? I see it being used below, but doesn't it belong here?

Reply:
@shaneutt  KTF is using helm charts deploy kong enterprise, according to the charts documents

Reply:
This doesn't seem clear, regardless of what the contents of `a.license` is supposed to be, the problem seems to remain that we're _simply not using it_.

Reply:
If user specified, it is used here
https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R189